tools/nnyeah/nnyeah/Program.cs

```diff
@@ -1,23 +1,86 @@
 using System;
 using System.IO;
+using System.Reflection;
+using Mono.Options;
+using System.Collections.Generic;
 
 #nullable enable
 
 namespace nnyeah {
     class Program {
         static void Main (string [] args)
         {
+            var doHelp = false;
+            string? infile = null, outfile = null;
+            var verbose = false;
+            var suppressWarnings = false;
+            var warnings = new List<string> ();
+            var transforms = new List<string> ();
+
+            var options = new OptionSet () {
+                { "h|?|help", o => doHelp = true },
+                { "i=|input=", f => infile = f },
+                { "o=|output=", f => outfile = f },
+                { "v|verbose", o => verbose = true },
+                { "s|suppress-warnings", o => suppressWarnings = true },
+            };
+
+            var extra = options.Parse (args);
+
+            if (doHelp || infile is null || outfile is null) {
+                PrintOptions (options, Console.Out);
+                Environment.Exit (0);
+			}
+
+
             if (args.Length != 2) {
```

Comment:
This logic breaks if passed -v, since there will be more than two arguments.

Reply:
old code, now gone.